src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Avx2.PlatformNotSupported.cs

```diff
@@ -252,6 +252,50 @@ public abstract class Avx2 : Avx
         /// </summary>
         public static Vector128<T> BroadcastScalarToVector128<T>(Vector128<T> value) where T : struct { throw new PlatformNotSupportedException(); }
 
+        /// <summary>
+        /// __m128i _mm_broadcastb_epi8 (__m128i a)
```

Comment:
nit: We've left comments elsewhere when the API exposed doesn't have a corresponding native API

Reply:
Thanks, will add.